src/controllers/dlgprefcontroller.cpp

```diff
@@ -21,6 +21,17 @@
 #include "preferences/usersettings.h"
 #include "util/version.h"
 
+QString nameForPreset(const PresetInfo& preset);
```

Comment:
hey @rryan when I put this in the nameForPreset in the namespace, it shows error, compiler error, ambiguity arises by this step, so better to put i need change every call of nameForPreset, so will it be better to put the function back to the global namespace